Makefile/Makefile.sublime-syntax

```diff
@@ -592,14 +593,18 @@ contexts:
       captures:
         1: meta.string.makefile string.unquoted.makefile
         2: meta.string.makefile meta.interpolation.makefile punctuation.section.interpolation.begin.makefile
-      embed: scope:source.shell
-      embed_scope: meta.string.makefile meta.interpolation.makefile
+      embed: embedded-shell
+      embed_scope: meta.string.makefile meta.interpolation.makefile source.shell.embedded
       escape: (?!<\\)\2
       escape_captures:
         0: meta.string.makefile meta.interpolation.makefile punctuation.section.interpolation.end.makefile
     - match: ''
       set: value-to-be-defined
 
+  embedded-shell:
+    - include: scope:source.shell#prototype
+    - include: scope:source.shell
```

Comment:
This should be able to use the new `apply_prototype: true` option instead of explicitly including the prototype context on the line above.